gramps/gen/utils/symbols.py

```diff
@@ -0,0 +1,409 @@
+                                    "&#10016;", "\u2720",  "+"),
+                 (_("Star of david")        +' : \u2721',
+                                    "&#10017;", "\u2721",  "+"),
+                 (_("Dead"),
+                                   _("Dead"),  _("Dead"), _("Dead"))
+                ]
+
+    def __init__(self):
+        self.symbols = None
+    #
+    # functions for general symbols
+    #
+    def get_symbol_for_html(self, symbol):
+        """ retun the html string like '&#9898;' """
+        return self.all_symbols[symbol][2]
+
+    def get_symbol_for_string(self, symbol):
+        """ retun the utf-8 character like '\u2670' """
+        return self.all_symbols[symbol][1]
+
+    def get_symbol_replacement(self, symbol):
+        """
+        Return the replacement string.
+        This is used if the utf-8 symbol in not present within a font.
+        """
+        return self.all_symbols[symbol][3]
+
+    #
+    # functions for death symbols
+    #
+    def get_death_symbols(self):
+        """
+        Return the list of death symbols.
+        This is used in the global preference to choose which symbol we'll use.
+        """
+        return self.death_symbols
+
+    def get_death_symbol_name(self, symbol):
+        """
+        Return the name of the symbol.
+        The symbol correspond to the selected string for html which is saved
+        in the config section for interface.death-symbol
+        """
+        for element in self.death_symbols:
+            if element[1] == symbol:
+                return element[0]
+        return ""
+
+    def get_death_symbol_for_html(self, symbol):
+        """
+        return the html string like '&#9898;'.
+        Should be used only here for test.
+        """
+        return self.death_symbols[symbol][1]
+
+    def get_death_symbol_for_char(self, symbol):
+        """
+        Return the utf-8 character for the symbol.
+        """
+        return self.death_symbols[symbol][2]
+
+    def get_death_symbol_for_string(self, symbol):
+        """
+        Return the utf-8 character for the symbol.
+        The symbol correspond to the selected string for html which is saved
+        in the config section for interface.death-symbol
+        """
+        for element in self.death_symbols:
+            if element[1] == symbol:
+                return element[2]
+        return ""
+
+    def get_death_symbol_replacement(self, symbol):
+        """
+        Return the string replacement for the symbol.
+        The symbol correspond to the selected string for html which is saved
+        in the config section for interface.death-symbol
+        """
+        for element in self.death_symbols:
+            if element[1] == symbol:
+                return element[3]
+        return ""
+
+    #
+    # functions for all symbols
+    #
+    def get_how_many_symbols(self):
+        return len(self.death_symbols) + len(self.all_symbols) - 3
+
+if __name__ == '__main__':
```

Comment:
Is it easy to write a unit test instead?

Reply:
When I started to write the module, the tests was included.
I removed this code. I'll write a test for that. Can it be a standalone test or it will be included with all other tests.